### PR TITLE
Solorigate: Extract to separate qlpack

### PR DESCRIPTION
**Pull request by aeisenberg**
Extracts solorigate to separate qlpacks in preparation for publishing them to the registry.

### CODE REVIEW

**Comment by aeisenberg** [comment]
Solorigate queries are now available from the package registry. Run them like this:

```
codeql database analyze --format=sarif-latest --output=out --download <any-csharp-db> codeql/solorigate-queries
```

**Comment by aeisenberg** [comment]
This PR also includes a simple script to re-publish and version bump.

File: csharp/ql/campaigns/Solorigate/src/qlpack.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: codeql/solorigate-queries
```
Comment (hvitved) [inline csharp/ql/campaigns/Solorigate/src/qlpack.yml:1]: should this mention `csharp`?

File: csharp/ql/campaigns/Solorigate/lib/qlpack.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: codeql/solorigate-all
```
Comment (hvitved) [inline csharp/ql/campaigns/Solorigate/lib/qlpack.yml:1]: should this mention `csharp`?

File: csharp/ql/campaigns/Solorigate/src/qlpack.yml (hunk 1 above)
Comment (aeisenberg) [reply csharp/ql/campaigns/Solorigate/src/qlpack.yml:1]: Good point. I made the change. Unpublished previous packs and re-published under the new name.

Comment (hvitved) [reply csharp/ql/campaigns/Solorigate/src/qlpack.yml:1]: Did you forget to push the change?

Comment (aeisenberg) [reply csharp/ql/campaigns/Solorigate/src/qlpack.yml]: I pushed to the wrong remote. Should be good now.

File: csharp/ql/campaigns/Solorigate/test/qlpack.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: csharp-solorigate-tests
+groups:
+    - csharp
+    - solorigate
+    - test
+dependencies:
+  codeql/csharp-all: "*"
+  codeql/csharp-queries: "*"
+  codeql/solorigate-all: "*"
+  codeql/solorigate-queries: "*"
```
Comment (hvitved) [inline csharp/ql/campaigns/Solorigate/test/qlpack.yml:10]: These should be updated, right?

Comment (aeisenberg) [reply csharp/ql/campaigns/Solorigate/test/qlpack.yml]: Yes. Done.

**Review by hvitved — APPROVED** [review]

**Comment by aeisenberg** [comment]
Thanks @hvitved for reviewing!

**Pull request merged**